conda-recipes/iqsharp/build.ps1

```diff
@@ -14,8 +14,11 @@ if ($IsWindows) {
 }
 
 # Find the repo root relative to this script.
-$RepoRoot = Resolve-Path (Join-Path $PSScriptRoot ".." "..");
+$RepoRoot = Resolve-Path (Join-Path $PSScriptRoot "../..");
```

Comment:
As of PowerShell Core (6.0), `Join-Path` takes multiple parameters for each child path to be joined. Was this change added to resolve a Windows PowerShell vs PowerShell Core breakage?

Reply:
yes.